src/ethereum/testnets/clique.py

```diff
@@ -0,0 +1,135 @@
+"""
+Clique
+^^^^^^
```

Comment:
Might be nice to expand the heading a bit for people not familiar with clique?

```suggestion
Clique Consensus
^^^^^^^^^^^^^^^^
```